packages/react/src/components/Tabs/Tabs.props.ts

```diff
@@ -0,0 +1,15 @@
+import { IconProps } from "../Icon/Icon.props";
+
+export interface TabsProps {
+  /**
+   * Specify the items inside a tab
+   */
+  items: Required<Array<TabItem>>;
+}
+
+export interface TabItem {
+  icon?: IconProps;
+  label: Required<string>;
+  component: Required<string>;
```

Comment:
Why not pass in the component here directly? The component could then be placed directly in the content area.
content: <Text value="something" />

Reply:
@bashlk I think he has done this to keep it in sync with twig, I can do the change but that would require the change to be done in twig as well which could be a breaking change. What do you think ? Cc :- @justintemps

Reply:
Also did the syncup to develop and made a few changes (Added them above)

Reply:
In this case, I think reworking the props to accept the component directly is better since it leads to a simpler implementation. It does mean that the props to the component will be slightly different than the twig version but here we are leveraging one of the features in React that makes it easier to achieve this kind of component composition. I am not sure whether we achieve the same pattern in twig in a nice way.

Reply:
Agree with @bashlk. In terms of keeping the two libs in sync, the React and Twig components don't necessarily have to take the exact same props. They do need to share the same stylesheets and we want to avoid styles specific to one implementation or the other. But we shouldn't feel like we have to implement the React components in a Twig way. React is React and when we can take advantage of it to simplify things, then we should, even if that makes the underlying logic different than the logic of the Twig component.

Reply:
Got it guys will do the change thanks :)